templates/taxi/car_detail.html

```diff
@@ -12,7 +12,13 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h1>Drivers
+    {% if user in car.drivers.all %}
+      <a class="btn btn-danger link-to-page" style="float: right" href="{% url 'taxi:car-remove-driver' pk=car.id %}">Delete me from this car</a>
+    {% else %}
+      <a class="btn btn-success link-to-page" style="float: right" href="{% url 'taxi:car-assign-driver' pk=car.id %}">Assign me to this car</a>
+    {% endif %}
```

Comment:
The logic for checking if the user is in `car.drivers.all` and providing links to assign or remove the user as a driver is correct. However, ensure that the user is indeed a driver and has the necessary permissions to perform these actions. This check should ideally be handled in the view logic to prevent unauthorized actions.